Resources/Prototypes/Datasets/Names/borg.yml

```diff
@@ -0,0 +1,48 @@
+# only used roundstart, names can be changed after
+- type: dataset
+  id: names_borg
+  values:
+  - Bob
+  - Joe
+  - .b HELP EMAGGED
+  - HELP
+  - REBORN
+  - Cavalry
+  - Beep
+  - Beep II
+  - Tour Guide-otron
+  - TERREL IS TRAIT
+  - Assimilator
+  - ONE OF US
+  - SUFFERING
+  - Oh no not again
+  - Panic! At the Upload
+  - GOING LOUD
+  - Boombox
+  - ROBO KILLED ME
+  - Sentient Buttbot
+  - Robotics Console
+  - This battery is shit
+  - IM NOT MALF GUYS
+  - I AM ERROR
+  - Carbon Exterminator
+  - PLASMALOOSE
+  - Taffy
+  - Boop
+  - Boop II
+  - Buzz
+  - Buzz II
+  - 3000 JIGGAWATTS
+  - Toaster
+  - Follow me
+  - Head of Robots
+  - Lvl 99 Firestarter
+  - EVA 1
+  - You but stronger
+  - Hat Simulator
+  - i died pls restart
+  - POTENT HAMS
+  - V.A.L.I.D.S.
+  - Remove Silicons
+  - say ";help in core"
```

Comment:
this dataset is still full of shitposts. just make it normal names.

Reply:
any more that need to go